dissect/target/plugins/general/example.py

```diff
@@ -53,7 +53,8 @@ def __init__(self, target):
             super().__init__(target)
     """
 
-    __findable__ = False
+    __findable__: bool = False
+    __skip__: bool = True
```

Comment:
```suggestion
    # IMPORTANT: Remove these attributes when using this as boilerplate for your own plugin!
    __findable__ = False
    __skip__ = True
```
Don't think these are necessary? Also adding a small comment

Reply:
I was in doubt.